src/subworkflows/subworkflow.js

```diff
@@ -315,4 +315,12 @@ export class Subworkflow extends BaseSubworkflow {
         const index = this.units.findIndex((u) => u.flowchartId === id);
         return `units.${index}`;
     }
+
+    findUnitWithTag(tag) {
+        return this.units.find((unit) => unit.tags.includes(tag));
+    }
+
+    get hasConvergence() {
+        return !!this.convergenceParam && !!this.convergenceResult && !!this.convergenceSeries;
```

Comment:
`this.convergenceSeries` is not a getter, you would need to pass `scopeTrack` here

Reply:
used to ensure the function exists, not that there is a result, since the job's scopeTrack is not in scope here